lib/presentation/ui/home/home_page.dart

```diff
@@ -438,3 +530,104 @@ class _PermissionWidget extends StatelessWidget {
   }
 }
 
+class RemoveSelectedOccurrenceFloatingActionButton extends StatelessWidget {
+  final HomeBloc homebloc;
+  RemoveSelectedOccurrenceFloatingActionButton(
+      {Key key, @required this.homebloc})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: Alignment.bottomLeft,
+      child: Container(
+        margin: EdgeInsets.only(bottom: 5, left: 5),
+        child: FloatingActionButton(
+          onPressed: () {
+            homebloc.openOccurrenceSink.add(false);
+            homebloc.selectedOccurrenceSink.add(null);
+          },
+          child: Icon(Icons.clear),
+          backgroundColor: Colors.orange[300],
+        ),
+      ),
+    );
+  }
+}
+
+class GetOccurrenceFloatingActionButton extends StatelessWidget {
+  final HomeBloc homebloc;
+  final bool open;
+  GetOccurrenceFloatingActionButton(
+      {Key key, @required this.homebloc, @required this.open})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: Alignment.bottomRight,
+      child: Container(
+        margin: EdgeInsets.only(bottom: 5, right: 5),
+        child: FloatingActionButton(
+          onPressed: () {
+            homebloc.openOccurrenceSink.add(!open);
```

Comment:
So that I can understand, so this is a Stateless widget that always has the same bool value? And you're adding always that bool value to the sink?

So you create 2 of these, one for the open and one for the close state?